app/views/articles/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<body>
+<div class="main">
+  <div class="container">
+
+    <p>heyyyy</p>
```

Comment:
Well 'heyyyy' to you too. Was this intentionally left in?

Reply:
Leaving it in wasn't exactly intentional but it kind of suits the improvisational tone of the site 🙃